GitHubTeams.ps1

```diff
@@ -276,6 +315,488 @@ filter Get-GitHubTeamMember
+    {
+        $getGitHubTeamParms['AccessToken'] = $AccessToken
+    }
+    if ($PSBoundParameters.ContainsKey('NoStatus'))
+    {
+        $getGitHubTeamParms['NoStatus'] = $NoStatus
+    }
+
+    $team = Get-GitHubTeam @getGitHubTeamParms
+
+    $uriFragment = "/orgs/$OrganizationName/teams/$($team.slug)"
+
+    $hashBody = @{
+        name = $TeamName
+    }
+
+    if ($PSBoundParameters.ContainsKey('Description')) { $hashBody['description'] = $Description }
+    if ($PSBoundParameters.ContainsKey('Privacy')) { $hashBody['privacy'] = $Privacy.ToLower() }
+    if ($PSBoundParameters.ContainsKey('ParentTeamName'))
+    {
+        $getGitHubTeamParms = @{
+            OrganizationName = $OrganizationName
+            TeamName = $ParentTeamName
+            Whatif = $false
+            Confirm = $false
+        }
+        if ($PSBoundParameters.ContainsKey('AccessToken'))
+        {
+            $getGitHubTeamParms['AccessToken'] = $AccessToken
+        }
+        if ($PSBoundParameters.ContainsKey('NoStatus'))
+        {
+            $getGitHubTeamParms['NoStatus'] = $NoStatus
+        }
+
+        $team = Get-GitHubTeam @getGitHubTeamParms
+
+        $hashBody['parent_team_id'] = $team.id
+    }
+
+    $params = @{
+        UriFragment = $uriFragment
+        Body = (ConvertTo-Json -InputObject $hashBody)
+        Method = 'Patch'
+        Description =  "Updating $TeamName"
+        AccessToken = $AccessToken
+        TelemetryEventName = $MyInvocation.MyCommand.Name
+        TelemetryProperties = $telemetryProperties
+        NoStatus = (Resolve-ParameterWithDefaultConfigurationValue `
+            -Name NoStatus -ConfigValueName DefaultNoStatus)
+    }
+
+    return (Invoke-GHRestMethod @params | Add-GitHubTeamAdditionalProperties)
+}
+
+filter Remove-GitHubTeam
+{
+<#
+    .SYNOPSIS
+        Removes a team from an organization on GitHub.
+
+    .DESCRIPTION
+        Removes a team from an organization on GitHub.
+
+        The Git repo for this module can be found here: http://aka.ms/PowerShellForGitHub
+
+    .PARAMETER OrganizationName
+        The name of the organization the team is in.
+
+    .PARAMETER TeamName
+        The name of the team.
+
+    .PARAMETER Force
+        If this switch is specified, you will not be prompted for confirmation of command execution.
+
+    .PARAMETER AccessToken
+        If provided, this will be used as the AccessToken for authentication with the
+        REST Api.  Otherwise, will attempt to use the configured value or will run unauthenticated.
+
+    .PARAMETER NoStatus
+        If this switch is specified, long-running commands will run on the main thread
+        with no commandline status update.  When not specified, those commands run in
+        the background, enabling the command prompt to provide status information.
+        If not supplied here, the DefaultNoStatus configuration property value will be used.
+
+
+    .INPUTS
+        GitHub.Team
```

Comment:
You are also supporting `GitHub.Organization`

Reply:
Done